docs/devctr-image.md

```diff
@@ -45,10 +45,21 @@ registry. The Firecracker CI suite must also be updated to use the new image.
 1. Commit the changes, if any.
 
 1. Build a new container image with the updated Dockerfile.
+   a: Additionally also checks for any outdated python packages
```

Comment:
Please add another newline here to render it properly in markdown

Reply:
ack